core/unit_test/TestCTestDevice.cpp

```diff
@@ -1,4 +1,7 @@
 #include <gtest/gtest.h>
+#ifndef KOKKOS_HAS_BUNDLED_GTEST
```

Comment:
minor: since we only check for **not** defined, it could be more readable as `#ifdef KOKKOS_HAS_EXTERNAL_GTEST`